docs/config.md

```diff
@@ -1950,8 +1949,9 @@ Listen Addresses:
 
 #### `Swarm.Transports.Network.QUIC`
 
-[QUIC](https://en.wikipedia.org/wiki/QUIC) is a UDP-based transport with
-built-in encryption and multiplexing. The primary benefits over TCP are:
+[QUIC](https://en.wikipedia.org/wiki/QUIC) is the most widely used transport by
+Kubo nodes. It is a UDP-based transport with built-in encryption and
+multiplexing. The primary benefits over TCP are:
 
 1. It doesn't require a file descriptor per connection, easing the load on the OS.
 2. It currently takes 2 round trips to establish a connection (our TCP transport
```

Comment:
Also I would add HOL to that list.

Reply:
Also the 2 rounds trips is wrong. Just 1.